rosys/vision/mjpeg_camera/motec_mjpeg_device.py

```diff
@@ -0,0 +1,45 @@
+from typing import Optional
+
+from .mjpeg_device import MjpegDevice
+from .motec_settings_interface import MotecSettingsInterface
+from .vendors import VendorType, mac_to_vendor
+
+
+class MotecMjpegDevice(MjpegDevice):
+    def __init__(self, mac: str, ip: str, *,
+                 username: Optional[str] = '',
+                 password: Optional[str] = '',
+                 control_port: Optional[int] = 8885) -> None:
+
+        super().__init__(mac, ip,
+                         username=username, password=password)
+
+        vendor = mac_to_vendor(mac)
+        if not vendor == VendorType.MOTEC:
```

Comment:
I like it more this way, both is fine.
```suggestion
        if vendor != VendorType.MOTEC:
```